lib/zipkin-tracer/config.rb

```diff
@@ -39,6 +39,10 @@ def initialize(app, config_hash)
       if @sampled_as_boolean
         @logger && @logger.warn("Using a boolean in the Sampled header is deprecated. Consider setting sampled_as_boolean to false")
       end
+      # Record the given tags on server receive, even if the zipkin headers were present in the incoming request?
+      tags = config[:record_on_server_receive]
+      @record_on_server_receive = present?(tags) ? parse_record_on_server_receive(tags) : nil
```

Comment:
I would do `{}` instead of `nil` and later you do not need to check for `nil` just do `.each` on the hash